nodes/config.html

```diff
@@ -214,6 +224,11 @@ <h3>Details</h3>
     <p>Server field must not contains the scheme (<code>http://</code> or <code>https://</code>).</p>
     <p>The <code>Webhook Section</code> allow to configure the URL on which the Hubitat hub will send events.
       Ensure that Node-RED is reachable by the Hubitat hub</p>
+    <p>
+    <p><b>Rebuild cache on Hubitat <code>systemStart</code> event</b> implies to request hub to re-initialize hubitat's nodes.
+      If desynchronization is found, a new event for each desynchronized attribute will be sent.
+      The event generated cannot be exactly like an original event. Consider that only <code>value</code> as a valid property.
+      The <code>systemStart</code> event is triggered when Hubitat start.</p>
```

Comment:
@Botched1 do you want to review/improve/clarify this documentation about systemStart event?

Reply:
```suggestion
    <p>
    <p><b>Rebuild cache on Hubitat <code>systemStart</code> event</b> will rebuild the Node-RED cache for the Hubitat nodes.
      During the rebuild, if a difference is found between the current Hubitat attribute value and the previous cached value a new event msg for each changed attribute will be sent from the applicable nodes.
      If the current Hubitat attribute value is unchanged from the previous cached value, then no event msg will be sent from the rebuild.
      The <code>systemStart</code> event is triggered any time a Hubitat hub starts up after a power cycle or reboot.</p>
```

Reply:
Thanks, really appreciated :+1: